PowerEditor/src/Parameters.cpp

```diff
@@ -1939,9 +1943,442 @@ bool NppParameters::getUserStylersFromXmlTree()
 {
 	TiXmlNode *root = _pXmlUserStylerDoc->FirstChild(L"NotepadPlus");
 	if (!root) return false;
+	updateUserStylersFromModelXml(root);
 	return feedStylerArray(root);
 }
 
+void NppParameters::updateKeyWordsFromModelXml(TiXmlNode* rootUser)
+{
+	std::wstring modelLangsPath(_nppPath);
+	pathAppend(modelLangsPath, L"langs.model.xml");
+	TiXmlDocument* pXmlModel = new TiXmlDocument(modelLangsPath);
+	int buffer_size = WideCharToMultiByte(CP_ACP, 0, modelLangsPath.c_str(), static_cast<int>(modelLangsPath.length()), nullptr, 0, nullptr, nullptr);
+	std::string sModelPath(buffer_size, 0);
+	WideCharToMultiByte(CP_ACP, 0, modelLangsPath.c_str(), static_cast<int>(modelLangsPath.length()), sModelPath.data(), buffer_size, nullptr, nullptr);

```

Comment:
Use wstring2string() will simplify the code.

Reply:
I updated to use wstring2string() and string2wstring() for those.